msi_perkeyrgb/main.py

```diff
@@ -126,7 +130,36 @@ def main():
             # If user has requested to display a steady color
             elif args.steady:
                 try:
```

Comment:
Just an idea but to make things more consistent with other applications, we may want to only try to parse it as a hex if the color is preceded by a `#` instead of using an try block. Then we could use an if statement to check to check whether it's a hex or a color. This way we can better give feedback to the user whether they typed invalid hex or an invalid color name plus cleans up the code a bit.

Reply:
Good suggestion but it makes the usage syntax inconsistent with previous versions, which may be a slight nuisance to any existing users that update to the latest version. Do you think it's important enough to change?

Reply:
To be honest I don't know and is probably something which @Askannz needs to decide on.

Reply:
Sounds good. Leaving it up to @Askannz for now.